src/EpicWithHeaderImage/index.stories.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { ReactElement } from 'react';
+import { StorybookWrapper } from '../utils/StorybookWrapper';
+import { BrazeEndOfArticleComponent } from '../BrazeEndOfArticleComponent';
+import { knobsData } from '../utils/knobsData';
+import {
+    coreArgTypes,
+    ophanComponentIdArgType,
+    buildEpicParagraphDocs,
+} from '../storybookCommon/argTypes';
+import { BrazeMessageProps } from '../Epic';
+
+const NUMBER_OF_PARAGRAPHS = 9;
+const paragraphDocs = buildEpicParagraphDocs(NUMBER_OF_PARAGRAPHS);
+
+export default {
+    component: 'EpicWithHeaderImage',
+    title: 'WorkInProgress/EndOfArticle/EpicWithHeaderImage',
+    argTypes: {
+        ...coreArgTypes,
+        ...ophanComponentIdArgType,
+        heading: {
+            name: 'heading',
+            type: { name: 'string', required: true },
+            description: 'Header text',
+        },
+        ...Object.fromEntries(paragraphDocs),
+        highlightedText: {
+            name: 'highlightedText',
+            type: { name: 'string' },
+            description: 'Yellow highlighted text at end of final paragraph. Supports HTML.',
+        },
+        buttonText: {
+            name: 'buttonText',
+            type: { name: 'string', required: true },
+            description: 'Button label text',
+        },
+        buttonUrl: {
+            name: 'buttonUrl',
+            type: { name: 'string', required: true },
+            description:
+                'Destination link for the button. See Braze user guide for tracking params.',
+        },
+    },
+};
+
+const StoryTemplate = (
+    args: BrazeMessageProps & { componentName: string },
+): ReactElement | null => {
+    const brazeMessageProps = {
+        ophanComponentId: args.ophanComponentId,
+        heading: args.heading,
+        highlightedText: args.highlightedText,
+        buttonText: args.buttonText,
+        buttonUrl: args.buttonUrl,
+        paragraph1: args.paragraph1,
+        paragraph2: args.paragraph2,
+        paragraph3: args.paragraph3,
+        paragraph4: args.paragraph4,
+        paragraph5: args.paragraph5,
+        paragraph6: args.paragraph6,
+        paragraph7: args.paragraph7,
+        paragraph8: args.paragraph8,
+        paragraph9: args.paragraph9,
+    };
+
+    knobsData.set({ ...brazeMessageProps, componentName: args.componentName });
+
+    return (
+        <StorybookWrapper>
+            <BrazeEndOfArticleComponent
+                brazeMessageProps={brazeMessageProps}
+                componentName={args.componentName}
+                subscribeToNewsletter={() => Promise.resolve()}
+            />
+        </StorybookWrapper>
+    );
+};
+
+export const DefaultStory = StoryTemplate.bind({});
+
+DefaultStory.args = {
```

Comment:
Should this default story include header text? II guess it's missing because it's hard-coded in the template, in which case that makes sense.

Reply:
Yeah that's correct. The original/core epic template support supports a simple text header which is provided dynamically from Braze. Here we're omitting it as we're using the hardcoded new special header section.